packages/react-cookie/src/server.ts

```diff
@@ -0,0 +1,12 @@
+import {Context} from 'koa';
```

Comment:
I still think the way these two packages are broken up is strange. This function in particular is strange when the rest of the server cookie stuff is just part of react-network. Why can't this just be part of `react-network`'s `applyToContext`?

Reply:
It can be 👍 Slightly related, I am considering pulling the shared types and server manager into a more generic `@shopify/cookie` package that `react-network` and `react-cookie` would depend on.